api/v1beta1/validations.go

```diff
@@ -118,6 +123,146 @@ func ValidateCount(count *intstr.IntOrString) error {
 	return nil
 }
 
+func newGoValidator() (*validator.Validate, ut.Translator, error) {
```

Comment:
WDYT of splitting this function in multiple functions to reduce its size to facilitate its readability?